src/commands/createComponent.ts

```diff
@@ -19,11 +20,13 @@ export async function createComponent(
   }
 
   const targetPath = workspace.workspaceFolders[0].uri.toString().replace("file://", "");
-  const outFile = `${targetPath}/${name}`;
+  const outFile = `${targetPath}/component-archive`;
 
```

Comment:
can we  add a check here to see if there is already a `component-archive` directory and if so prompt the user to confirm that they wish to overwrite?